desc/compute/utils.py

```diff
@@ -845,9 +777,8 @@ def surface_integrals_map(grid, surface_label="rho", expand_out=True):
     # previous paragraph.
     masks = cond(
         has_endpoint_dupe,
-        lambda masks: put(masks, jnp.asarray([0, -1]), masks[0] | masks[-1]),
-        lambda masks: masks,
-        masks,
+        lambda: put(masks, jnp.array([0, -1]), masks[0] | masks[-1]),
+        lambda: masks,
```

Comment:
My old benchmark script for this showed there was no performance difference in closing over this boolean array